drivers/driver_linux.go

```diff
@@ -130,6 +131,10 @@ func GetFSMagic(rootpath string) (FsMagic, error) {
 	if err := unix.Statfs(filepath.Dir(rootpath), &buf); err != nil {
 		return 0, err
 	}
+
+	if _, ok := FsNames[FsMagic(buf.Type)]; ok != true {
```

Comment:
It's more idiomatic to use `!ok` here.  Please include the pathname of the directory we were looking at in the debug message.

Reply:
Understood and will do.